### PR TITLE
fix: environment and collection sync issue with firebase

### LINKED ISSUE #900: Collections & Environments not synced
**Describe the bug**
My collections & environments not synced, even after toogling sync option in setting. Only my request histories were synced.

**To Reproduce**
1. Open postwoman
2. Signin with google
3. Import collections
4. Toggling on sync option
5. Try to open postwoman in incognito and in my smartphone, logged in with google
6. Got blank collections & environments, only my histories were synced

**Expected behavior**
All collections & environments are synced

**Desktop:**
 - OS: Manjaro
 - Browser Chromium
 - Version 81.0.4044.138

**Additional info**
I use postwoman v1.9.7, where [#854](https://github.com/liyasthomas/postwoman/pull/854) already merged

### PR DESCRIPTION
**Pull request by myussufz**
Fixes #900

### CODE REVIEW

**Comment by liyasthomas** [comment]
@myussufz thanks for the quick PR, I'll merge it ASAP after running some tests.

**Comment by myussufz** [comment]
@liyasthomas I push more changes to fix the sync with firebase, so it able to sync with firebase more often, you may try to check whether it works as expected or not.

File: components/collections/addCollection.vue [hunk 1]
```diff
@@ -78,6 +78,7 @@ export default {
     },
     hideModal() {
       this.$emit("hide-modal")
+      this.$data.name = undefined
```
Comment (liyasthomas) [inline components/collections/addCollection.vue:81]: @myussufz can you tell me what does this LOC do?

Comment (myussufz) [reply components/collections/addCollection.vue:81]: I realize, every time I add a new collection, the input field keep the state, so I make it empty, every time the modal is hidden

**Pull request merged**